library/src/morpheus-master/sass/modules/navigation/_base.scss

```diff
@@ -1277,3 +1277,34 @@ body.is-logged-out{
         }
     }
 }
+
+@media only screen and (min-width: 801px) {
+	.manage-overview-link {
+		border: 1px solid #ccc;
+		color: #333 !important;
+		cursor: pointer;
+		display: inline-block;
+		font-family: "Open Sans", sans-serif;
+		font-size: 12px;
+		text-decoration: none !important;
+		-moz-appearance: none;
+		-webkit-appearance: none;
+		-moz-border-radius: 3px;
```

Comment:
There is a mixin' for border-radius: @include border-radius(3px); that will append the vendor prefixes.